### DISCUSSION TITLE
Application Trouble

### DISCUSSION THREAD

**Discussion by PhlompfDev** [community/community#131757] [GitHub Education]
### Hi everyone,

Hey y'all, 

I'm a virtual student who's not based near my campus. I've applied in the past and had trouble, but I quite recently applied again and was instantly rejected. I was rejected for poor image quality, but this is because the image was terribly compressed. The screen space given with the screen recorder is incredibly small and not enough for the proof I have. 

In my images I provided my financial aid as well as my full profile on my schools dashboard. I do not have a student ID because they do not send physical or virtual copies to off-campus students as there is technically no need. I have other means to prove my attendance, but this process is getting very tiring. 

Thank you for your time,

Phlompf

**Comment by samus-aran** [comment]
Hi @PhlompfDev 👋 

- [I can only use my Front Camera to take a photo.](https://github.com/orgs/community/discussions/111352#use-front-camera)

> We understand that students may not have access to high-quality cameras (laptop or phones) but as long as the text is legible the system should be able to process it. Here are a few tips on using a phone to capture.

1. Please ensure that you have allowed camera access on the browser you're using to apply. The steps for this will vary depending on which browser you are using. Information on browsers supported by GitHub can be found [here](https://docs.github.com/en/get-started/using-github/supported-browsers).
2. If you can, print off your evidence and take a photo of this copy.
3. Ensure your image is legible. If you are unable to read the text on the image itself then the verification process may not be able to either. If you are taking a photo of a screen try to have it as large a resolution as possible - take a photo of the evidence on a monitor rather than a phone.
4. Keep the text legible. It's preferred to have the full shot of the evidence but try and capture the important sections of it to confirm your student status. If the photo is too large and makes the text tiny then zoom in appropriately.
5. Alternatively you may use another mobile device or computer.

We do prefer live capture, not screenshots. Please bare that in mind. Let us know how you get on again.
